vendor.conf

```diff
@@ -6,7 +6,7 @@ github.com/containernetworking/cni v0.7.0-alpha1
 github.com/containers/image v1.3
 github.com/boltdb/bolt v1.3.1
 github.com/containers/libpod v1.0
-github.com/containers/storage v1.5
+github.com/containers/storage af01fe098c3eda83341f7c0f764956090efc5170 https://github.com/rhatdan/storage
```

Comment:
I'm assuming this will move to containers/storage before being merged?

Reply:
yup